packages/ember-application/lib/system/resolver.js

```diff
@@ -117,11 +117,9 @@ export default EmberObject.extend({
   init() {
     this._parseNameCache = dictionary(null);
   },
+
   normalize(fullName) {
-    let [
-      type,
-      name
-    ] = fullName.split(':', 2);
+    let [ type, name ] = fullName.split(':');
```

Comment:
I think it generates alright code https://cl.ly/2X0E1Z0Z163s/Screen%20Shot%202017-07-14%20at%209.46.10%20AM.jp2